app/main/views/brief_responses.py

```diff
@@ -214,6 +214,13 @@ def list_brief_responses():
     if brief_id is not None:
         brief_responses = brief_responses.filter(BriefResponse.brief_id == brief_id)
 
+    if request.args.get('framework_slug'):
+        brief_responses = brief_responses.join(BriefResponse.brief).join(Brief.framework).filter(
+            Brief.framework.has(Framework.slug.in_(
+                framework_slug.strip() for framework_slug in request.args["framework_slug"].split(",")
```

Comment:
So searching the API views for `request.args.get('framework')` I find 4 routes.

Two of them take comma separated framework slugs
One only supports a single framework slug
One filters on `framework.framework` rather than the slug

I'm going to go with the rough consistency therefore of `framework` rather than `framework_slug` but then it feels like we need to fix the `framework.framework` filtering. I will raise in the tech debt meeting